lib/sntools.js

```diff
@@ -223,18 +227,13 @@ class SNTools {
       // Try to get note title
       let title;
       try {
-        title = element.getElementsByTagName('title')[0].innerText;
+        title = element.getElementsByClassName('title')[0].innerHTML;
       } catch (e) {
         // Invalid note, continue
         console.log(note.name, 'is an invalid note (no title)');
         continue;
       }
 
-      // Check if title is date (default if no title is set). If so, use empty string
-      if (title !== '' && !isNaN(new Date(title))) {
```

Comment:
Is this no longer needed? What does EN do with as default title?

Reply:
I added fallback values for both EN and GK titles. It's better than having a note with a blank title or dates as titles IMO.